lib/multicell_location/services/polte_integration.c

```diff
@@ -0,0 +1,350 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr.h>
+#include <stdio.h>
+#include <modem/lte_lc.h>
+#include <modem/modem_info.h>
+#include <net/rest_client.h>
+#include <cJSON.h>
+#include <cJSON_os.h>
+
+#include "location_service.h"
+
+#include <logging/log.h>
+
+LOG_MODULE_REGISTER(multicell_location_polte, CONFIG_MULTICELL_LOCATION_LOG_LEVEL);
+
+#define API_KEY		CONFIG_MULTICELL_LOCATION_POLTE_API_TOKEN
+#define HOSTNAME	CONFIG_MULTICELL_LOCATION_HOSTNAME
+
+/* The timing advance returned by the nRF9160 modem must be divided by 16
+ * to have the range expected by Skyhook.
+ */
+#define TA_DIVIDER	16
```

Comment:
What is the expected timing advance unit with Polte? Modem reports the timing advance in 1 Ts, but Skyhook expects the units to be 16 Ts. If the same division is needed also with Polte, the comment should be updated (remove "Skyhook").